src/test/java/io/vertx/mqtt/test/server/MqttServerBadClientTest.java

```diff
@@ -39,18 +43,23 @@
 import io.vertx.ext.unit.Async;
 import io.vertx.ext.unit.TestContext;
 import io.vertx.ext.unit.junit.VertxUnitRunner;
+import io.vertx.mqtt.MqttClientOptions;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
+import static io.netty.handler.codec.mqtt.MqttQoS.AT_MOST_ONCE;
```

Comment:
It's better to use `import io.netty.handler.codec.mqtt.MqttQoS;` and then `MqttQoS.AT_MOST_ONCE` instead of just AT_MOST_ONCE